docs/architecture/adr-029-fee-grant-module.md

```diff
@@ -3,6 +3,7 @@
 ## Changelog
 
 - 2020/08/18: Initial Draft
+- 2021/05/05: Update `FeeAllowanceI` interface and fee allowances.  
```

Comment:
could you be more specific? ie add the fact that we removed height support and simplified naming